### PR TITLE
Add Tailchat to Communication - Custom Communication Systems

### PR DESCRIPTION
**Pull request by moonrailgun**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding has working installation instructions.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -370,6 +370,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 - [StoneAge Messenger](https://cweb.gitlab.io/StoneAge.html) - A self-hosted Android messenger, S3-compatible storage is the only backend needed. ([Source Code](https://gitlab.com/cweb-repos/cweb-conversations), [Clients](https://f-droid.org/en/packages/com.cweb.messenger/)) `GPL-3.0` `Java`
 - [Synapse](https://matrix.org/docs/projects/server/synapse) - Server for [Matrix](https://matrix.org/), an open standard for decentralized persistent communication. ([Source Code](https://github.com/matrix-org/synapse)) `Apache-2.0` `Python`
 - [Syndie](https://syndie.de) - Syndie is a libre system for operating distributed forums. `CC0-1.0` `Java`
+- [Tailchat](https://tailchat.msgbyte.com/) - Next generation noIM application in your own workspace, not only another Slack/Discord/Rocke.Chat. `Apache-2.0` `Nodejs`
```
Comment (Rabenherz112) [inline README.md:373]:
```suggestion
- [Tailchat](https://tailchat.msgbyte.com/) - Next generation noIM application in your own workspace, not only another Slack/Discord/Rocke.Chat. ([Demo](https://nightly.paw.msgbyte.com/), [Source Code](https://github.com/msgbyte/tailchat)) `Apache-2.0` `Docker`
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]
I noticed that the only installation method mentioned in your documentation is Docker, as outlined in the guide available at https://tailchat.msgbyte.com/docs/deployment/quickstart. Consequently, I changed the Language.

In addition, I included a link to your official demo, which corresponds to the one referenced in the quickstart guide as well as added your GitHub repository as the source code for the project.

File: README.md (hunk 1 above)
Comment (fbartels) [reply README.md:373]: Typo rocke.chat instead of rocket.chat

**Comment by moonrailgun** [comment]
Well, I have other way like `Compile from source` in https://tailchat.msgbyte.com/docs/deployment/docker-compose. But you are right because of Tailchat suggest user deploy with docker.

A other question is i notice that some repo will marked as docker/k8s. Tailchat also provide k8s config in https://tailchat.msgbyte.com/docs/deployment/kubernetes/simple. should i change to `Docker/K8S` ?

**Comment by moonrailgun** [comment]
and if i provide manual deploy guide, can i create new pr to change it to `Nodejs/Docker/K8S` or others

**Comment by nodiscc** [comment]
> and if i provide manual deploy guide, can i create new pr to change it to Nodejs/Docker/K8S or others

The language/platform tag should reflect the available installation methods, in order of preference. If your documentation recommends installing through Docker, then through K8S as an alternative, then by building from source/npm as a last resort, the language tag should be `Docker/K8S/Nodejs`

**Comment by moonrailgun** [comment]
Well, Thanks for your reply. I am plan to add deploy guide with manual(nodejs) next week.

**Pull request merged**